app/assets/javascripts/initializers/runtime.js

```diff
@@ -20,16 +20,13 @@ class Runtime {
   /**
    * This function returns a string that represents the current Medium where
    * the app is currently running. The currently supported mediums are Browser,
-   * ForemWebView and PWA.
+   * and ForemWebView.
    *
    * @returns {String} One of the supported Mediums or 'Unsupported'
```

Comment:
It's not related to this PR, but it's not correct at the moment for the logic we have 😅 

```suggestion
   * @returns {String} One of the supported Mediums 
```

Reply:
I'll sneak it in.